website/source/docs/agent/config_entries.html.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: "docs"
+page_title: "Configuration Entry Definitions"
+sidebar_current: "docs-agent-cfg_entries"
+description: |-
+  Consul allows storing configuration entries centrally to be used as defaults for configuring other aspects of Consul.
+---
+
+# Configuration Entries
+
+Configuration entries can be created to provide cluster-wide defaults for various aspects of Consul. Every configuration
+entry has at least two fields: `Kind` and `Name`. Those two fields are used to uniquely identify a configuration entry.
+When put into configuration files, configuration entries can be specified as HCL or JSON objects.
+
+Example:
+
+```hcl
+Kind = "<supported kind>"
+Name = "<name of entry>"
+```
+
+The two supported `Kind` configuration entries are detailed below.
+
+## Configuration Entry Kinds
+
+### Proxy Defaults - `proxy-defaults`
+
+Proxy defaults allow for configuring global config defaults across all services for Connect proxy configuration. Currently,
+only one global entry is supported.
+
+```hcl
+Kind = "proxy-defaults"
+Name = "global"
+Config {
+   local_connect_timeout_ms = 1000
+   handshake_timeout_ms = 10000
+}
+```
+
+* `Kind` - Must be set to `proxy-defaults`
+
+* `Name` - Must be set to `global`
+
+* `Config` - An arbitrary map of configuration values used by Connect proxies. See
+
+#### Proxy Configuration References
+
+* [Consul's Builtin Proxy](/docs/connect/configuration.html#built-in-proxy-options)
+* [Envoy](/docs/connect/proxies/envoy.html#bootstrap-configuration)
+
+### Service Defaults - `service-defaults`
+
+Service defaults control default global values for a service, such as its protocol.
+
+```hcl
+Kind = "service-defaults"
+Name = "web"
+Protocol = "http"
+```
+
+* `Kind` - Must be set to `service-defaults`
+
+* `Name` - Set to the name of the service being configured.
+
+* `Protocol` - Sets the protocol of the service. This is used by Connect proxies for things like observability features.
+
+## Applying Configuration Entries
+
+There are two ways to introduce new configuration entries to Consul. The first way is to use either the [API](/api/config.html) or [CLI](/docs/commands/config.html) to manage
+them in a running cluster. The second way is by placing inlined configuration entry definitions into the Consul server's
+[configuration file](/docs/agent/options.html#config_entries_bootstrap).
```

Comment:
nit: can be done later but I almost want to point not even talk about "two ways". But rather "You manage these via API, also as a convenience for initial cluster bootstrap there's this other thing". Do you think it's worth phrasing it that way now?